src/components/DateTimePicker.js

```diff
@@ -145,87 +148,96 @@ const DatePicker = React.createClass({
 
   render () {
     const styles = this.styles();
+    const inputSize = this.props.children ? 5 : 6;
 
     return (
-      <div style={styles.component}>
-        <div
-          onBlur={this._handleDateBlur}
-          onClick={this._handleDateClick}
-          onFocus={this._handleDateFocus}
-          ref='dateSelect'
-          style={Object.assign({}, styles.selectWrapper, this.state.showCalendar ? styles.activeSelectWrapper : null)}
-          tabIndex={0}
-        >
-          {this.props.showIcons ? (
-            <Icon
-              size={20}
-              style={styles.selectedIcon}
-              type={this.props.dateIcon}
-            />
-          ) : null}
-          <div style={styles.selectedText}>
-            {this.props.selectedDate ? moment.unix(this.props.selectedDate).format(this.props.dateFormat) : this.props.datePlaceholder}
-          </div>
-          <Icon
-            size={20}
-            style={styles.selectedDateCaret}
-            type={this.state.showCalendar ? 'caret-up' : 'caret-down'}
-          />
-        {this.state.showCalendar ? (
-          <Calendar
-            {...this.props}
-            onDateSelect={this._handleDateSelect}
-            style={styles.calendar}
-          />
-        ) : null}
-        </div>
-        <div>
-          {this.props.children}
-        </div>
-        <div
-          onBlur={this._handleTimeBlur}
-          onFocus={this._handleTimeFocus}
-          style={Object.assign({}, styles.selectWrapper, this.state.editTime ? styles.activeSelectWrapper : null)}
-          tabIndex={0}
-        >
-          {this.props.showIcons ? (
-            <Icon
-              size={20}
-              style={styles.selectedIcon}
-              type={this.props.timeIcon}
-            />
-          ) : null}
-          {this.state.editTime ? (
-            <input
-              autoFocus={true}
-              defaultValue={this.state.editTime ? moment.unix(this.props.selectedDate).format('HH:mm') : null}
-              name='time'
-              onBlur={this._handleTimeSelect}
-              ref='timeInput'
-              style={styles.timeInput}
-              type='time'
-            />
-          ) : (
-            <div style={styles.timeDisplay}>
-              {this.props.selectedDate ? moment.unix(this.props.selectedDate).format(this.props.timeFormat) : this.props.timePlaceholder}
-            </div>
-          )}
-          {this.props.timezoneFormat ? (
-            <div style={styles.timezone}>
-              {this._getTimezone(this.props.selectedDate)}
+      <Container fluid={true}>
+        <Row>
+          <Column span={{ large: inputSize, medium: inputSize, small: 12 }}>
```

Comment:
We're putting all of our span values in to a span function similar to the styles function.  Then you can use it the same way.  `const spans = this.spans();  spans.timeColumn`